src/Workspaces/Core/Portable/Rename/Renamer.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.CodeAnalysis.Rename
 {
     public static class Renamer
     {
+        internal static bool isRunningInUnitTests;
```

Comment:
This is concerning.  I think we *don't* today, but it might be nice to enable xunit to run unit tests in parallel.